assets/scripts/oppia-player-0.0.1.js

```diff
@@ -69,8 +69,8 @@
     function OppiaEmbed(oppiaNode) {
       // Validate the incoming node.
       if (oppiaNode.tagName !== 'OPPIA') {
-        throw ('Error: expected OppiaEmbed to be defined using a node of type ' +
-               '<oppia>, not ' + oppiaNode.tagName);
+        throw new Error('expected OppiaEmbed to be defined using ' +
```

Comment:
Per coding style guidelines, please break after '(' here and elsewhere if the contents span multiple lines.

Also, maybe change "expected" to "Expected" here and elsewhere, since it starts a sentence?

Reply:
Done.